src/3_PB/PianoDiProgetto/sections/PianificazioneSprint/NonoSprint.typ

```diff
@@ -10,7 +10,7 @@
   - Apportare modifiche e correzioni alla #glossary[documentazione] sulla base del feedback ricevuto nel corso della prima revisione #glossary[RTB], con particolare attenzione all'_Analisi dei Requisiti_;
   - Riprendere la stesura del _Piano di Progetto_:
     - Aggiornare pianificazione e preventivo pertinenti allo #glossary[sprint] 9;
-    - Aggiungere pianificazione e preventivo degli #glossary[sprint] 10 e 11, come parte della pianificazione a breve termine.
+    - Aggiungere pianificazione e preventivo degli #glossary[sprint] 10 e 11, come parte della pianificazione a breve termine;
```

Comment:
perche non "Aggiungere pianificazione dettagliata per gli sprint 10 e 11"? cioè non abbiamo una sezione chiamata pianificazione a breve termine

Reply:
questo è da considerare anche per le altre volte in cui compare un formato simile